packages/service-library/src/servicelib/aiohttp/long_running_tasks/_routes.py

```diff
@@ -18,37 +19,73 @@ class _PathParam(BaseModel):
     task_id: TaskId
 
 
+@routes.get("", name="list_tasks")
+async def list_tasks(request: web.Request) -> web.Response:
+    tasks_manager = get_tasks_manager(request.app)
+    task_context = get_task_context(request)
+    tracked_tasks: list[TrackedTask] = tasks_manager.list_tasks(
+        with_task_context=task_context
+    )
+
+    return web.json_response(
+        {
+            "data": [
+                TaskGet(
+                    task_id=task.task_id,
+                    task_name=task.task_name,
+                    status_href=f"{request.app.router['get_task_status'].url_for(task_id=task.task_id)}",
+                    result_href=f"{request.app.router['get_task_result'].url_for(task_id=task.task_id)}",
+                    abort_href=f"{request.app.router['cancel_and_delete_task'].url_for(task_id=task.task_id)}",
+                )
+                for task in tracked_tasks
```

Comment:
EXTRA MINOR: loop variable ``t`` reads better than ``task``  since the attribute also has the word task in it e.g. ``task_id=t.task_id`` vs ``task_id=task.task_id``